symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkAuthenticationConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.symphony.bdk.core.config.model;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class BdkAuthenticationConfig {
+
+  protected String privateKeyPath;
+  protected String certificatePath;
+  protected String certificatePassword;
```

Comment:
Following @symphony-youri's comment below, I also think that we should change our configuration model to: 
```yaml
bot:
    username: 'my-bot'
    rsa:
        privateKeyPath: 
        privateKeyContent: 
    certificate:
        path: 
        password: 
```

Reply:
but then should it be:

```
bot:
  username: 'my-bot'
  rsa:
    privateKey:
       path: / or content: 
  certificate:
    path: 
    password: 
```

😅

or maybe just rsa.path / rsa.content ? (and by the way is it really rsa or the notion of public/private keys? might be another algorithm)

so maybe just privateKey.path / privateKey.content?